newIDE/app/src/Utils/GDevelopServices/Shop.js

```diff
@@ -206,10 +206,11 @@ export const isProductAuthorizedResourceUrl = (url: string): boolean =>
   isPrivateAssetResourceAuthorizedUrl(url) ||
   isPrivateGameTemplateResourceAuthorizedUrl(url);
 
-export const extractFilenameWithExtensionFromProductAuthorizedUrl = (
-  url: string
+export const extractDecodedFilenameWithExtensionFromProductAuthorizedUrl = (
+  productAuthorizedUrl: string
 ): string => {
-  const urlWithoutQueryParams = url.split('?')[0];
+  const decodedUrl = decodeURIComponent(productAuthorizedUrl);
```

Comment:
Seems risky to use this function on a full URL. Consider doing it on the basename only

Reply:
oh yes, mb